x-pack/plugins/data_visualizer/public/application/file_data_visualizer/file_data_visualizer.tsx

```diff
@@ -38,7 +38,7 @@ export const FileDataVisualizer: FC<Props> = ({ getAdditionalLinks, resultLinks
     fieldFormats,
   };
 
-  const EmptyContext: FC = ({ children }) => <>{children}</>;
+  const EmptyContext = ({ children }: { children?: React.ReactNode }) => <>{children}</>;
```

Comment:
There's a bit of inconsistency with these ML changes, sometimes 
`const EmptyContext: FC = ({ children }) => ...`
Has been changed to:
`const EmptyContext = ({ children }: { children?: React.ReactNode }) => ...`

and sometimes it has been changed to:
`const EmptyContext: FC<{ children?: React.ReactNode }> = ({ children }) => ...`

I definitely preview the latter format. I can't see why some are different to others.

Reply:
It appears using `PropsWithChildren` is the more correct way to define these props, even in the situations where no other props are supplied. e.g. `FC<PropsWithChildren<{}>> ` for the above example